src/sql-parser/src/ast/mod.rs

```diff
@@ -1225,6 +1229,10 @@ impl fmt::Display for Statement {
                 f.write_str("SHOW CREATE SOURCE ")?;
                 write!(f, "{}", source_name)
             }
+            Statement::ShowCreateSink { sink_name } => {
+                f.write_str("SHOW CREATE SINK ")?;
+                write!(f, "{}", sink_name)
```

Comment:
You're definitely right to keep the consistency with the cases above, but `write!(f, "SHOW CREATE SINK {}", sink_name)` would be better! Don't actually fix this unless you feel like fixing it for all of these cases.